doc/contributing/submitting-changes.xml

```diff
@@ -77,6 +77,11 @@ Additional information.
           <command>firefox: 54.0.1 -> 55.0</command>
          </para>
         </listitem>
+        <listitem>
+         <para>
+          <command>onedrive: 2.3.13 ->! 2.4.2 (Notice the exclamation mark to be used when an update requires user intervention</command>
```

Comment:
I personally think that this syntax rather weird to read. How about sth like this: `package: 1.0.0 -> 2.0.0 (breaking)` (including further explanation in the release-notes and/or the commit-message)?

Apart from that, I'd suggest to gain more community feedback on this by e.g opening a Discourse thread.